packages/testing/src/mocking-activity-environment.ts

```diff
@@ -54,7 +54,7 @@ export class MockActivityEnvironment extends events.EventEmitter {
   /**
    * Run a function in Activity Context
    */
-  public async run<P extends any[], R, F extends ActivityFunction<P, R>>(fn: F, ...args: P): Promise<R> {
+  public async run<F extends ActivityFunction>(fn: F, ...args: Parameters<F>): ReturnType<F> {
```

Comment:
That's not compiling, because the compiler can't confirm that the return type of this async function is a `Promise`.

I think something like this should work:

```
public async run<F extends ActivityFunction>(fn: F, ...args: Parameters<F>): Promise<Awaited<ReturnType<F>>> {
```

Reply:
I just removed `async` since it doesn't use `await`.